tests/components/Microsoft.DotNet.UpgradeAssistant.Tests/Checks/MultiTargetingCheckTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.UpgradeAssistant.Checks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Tests.Checks
+{
+    public class MultiTargetingCheckTests
+    {
+        [Theory]
+        [MemberData(nameof(TestData))]
+        public async Task OnlyValidTFMsPassCheck(IProject[] projects, bool isReady)
+        {
+            // Arrange
+            var context = new Mock<IUpgradeContext>();
+            context.Setup(c => c.Projects).Returns(projects);
+
+            var readyCheck = new MultiTargetingCheck(NullLogger<MultiTargetingCheck>.Instance);
```

Comment:
I've been using `AutoMock` for this and it simplifies refactoings (ie if more services are injected into the check)

Reply:
This one's simple enough that I didn't really need AutoMock. I see what you mean about making future refactorings easy, though. I'll update it to use AutoMock.